File: Java-Core/src/main/java/presto/grammar/Specificity.java
package presto.grammar;

public enum Specificity {
	INCOMPATIBLE,
	RESOLVED,
	INHERITED,
	EXACT;
	
	public boolean greaterThan(Specificity other) {
		return new Integer(this.ordinal()).compareTo(new Integer(other.ordinal()))>0;
	}
}

File: Java-CodeServer/src/main/resources/js/prompto/grammar/Specificity.js
function Specificity(ordinal) {
	this.ordinal = ordinal;
	return this;
}

Specificity.INCOMPATIBLE = new Specificity(0);
Specificity.RESOLVED = new Specificity(1);
Specificity.INHERITED = new Specificity(2);
Specificity.EXACT = new Specificity(3);

Specificity.prototype.greaterThan = function(other) {
	return this.ordinal > other.ordinal;
}

exports.Specificity = Specificity;